index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">"Thank you for taking time to help us improve the platform."</p>
+    <form id = "survey-form">
+        <label id ="name-label">Name</label><input id = "name" type="text"  placeholder = "Write your name" required><br>
+        <label id ="email-label">Email</label> <input id="email" type="email" placeholder = "Write your email" required>
+        <label id="number-label">Age</label><input id="number" type="number" min = "5" max = "100" placeholder = "Enter age"></label>
```

Comment:
Please divide your work into sections. This helps to identify and group content that has a similar theme.